ui/scss/component/_claim-list.scss

```diff
@@ -133,6 +133,7 @@
   display: flex;
   flex-direction: column;
   flex: 1;
+  width: calc(100% - 12rem);
```

Comment:
Thanks for the PR! Where does the 12rem come from?

Reply:
Hi! 14rem is the fixed size of the thumbnail defined in _claim-list.scss.

https://github.com/lbryio/lbry-desktop/blob/b50ec8fab1ba0c3d7b09cb7fc8272bf41130b178/ui/scss/component/_claim-list.scss#L81-L83

If I subtract 14 rem, the space was too big, so I subtract 12rem.

Reply:
I think this should be possible without hardcoding some limit. That could cause issues when we add a better mobile view, or for larger claim previews.

Reply:
I think I need to use javascript to avoid hardcoding. Do you think it is a good idea?

Reply:
Not necessarily. Maybe check how much space the tags take up and then see if we should only render 3 tags instead of 4? (Or something like that).

It might be too much for the homepage to handle when you scroll down a lot and have a ton of tags on the page, but I'm all for trying it!